js_modules/dagster-ui/packages/ui-core/src/instance/backfill/BackfillAssetPartitionsTable.tsx

```diff
@@ -292,32 +302,39 @@ export const BACKFILL_PARTITIONS_FOR_ASSET_KEY_QUERY = gql`
 export function StatusBar({
   targeted,
   inProgress,
-  completed,
+  succeeded,
   failed,
 }: {
   targeted: number;
   inProgress: number;
-  completed: number;
+  succeeded: number;
   failed: number;
 }) {
+  const pctSucceeded = (100 * succeeded) / targeted;
+  const pctFailed = (100 * failed) / targeted;
+  const pctInProgress = (100 * inProgress) / targeted;
```

Comment:
Worth truncating the decimal value so you don't end up with super long decimal values?

Reply:
Ahh good call, I'll throw a .toFixed(4) on these when they're passed to the template grid CSS below just to keep it sane